packages/sanity/src/core/studio/tasks/components/TasksNavbarButton/TasksNavbarButton.tsx

```diff
@@ -0,0 +1,19 @@
+import {PanelRightIcon} from '@sanity/icons'
+import {Button} from '../../../../../ui-components'
+import {useTasksEnabled, useTasks} from '../../context'
+
+export function TasksNavbarButton() {
```

Comment:
Since this is a navbar specific component – should it be moved to the `/studio/components/navbar` folder?

Reply:
🤔 let's discuss it tomorrow in the daily, I do see some benefits on keeping all the tasks code in the same route but also theres a good point on keeping this into the navbar folder as the rest of the navbar buttons are there.